README.md

```diff
@@ -149,6 +153,8 @@ Moves the characters from `start` and `end` to `index`. Returns `this`.
 
 ### s.overwrite( start, end, content[, options] )
 
+__Note:__ It may be preferred to use `s.update(...)` instead
```

Comment:
Can we have a brief introduction of why?

Reply:
done